.github/workflows/integration.yml

```diff
@@ -88,17 +88,23 @@ jobs:
 
       - name: set fork job status
         uses: actions/github-script@v3
-        if: github.event_name == 'repository_dispatch'
+        if: ${{ always() }}
```

Comment:
why leave the if?

Reply:
GHA defaults to "success()", so this step wouldn't run if previous failed otherwise